src/main/java/com/devcycle/sdk/server/cloud/DVCCloudClient.java

```diff
@@ -22,7 +22,7 @@ public final class DVCCloudClient {
 
   private static final String DEFAULT_PLATFORM = "Java";
   private static final String DEFAULT_PLATFORM_VERSION = System.getProperty("java.version");
-  private static final User.SdkTypeEnum DEFAULT_SDK_TYPE = User.SdkTypeEnum.SERVER;
+  private static final PlatformData.SdkTypeEnum DEFAULT_SDK_TYPE = PlatformData.SdkTypeEnum.SERVER;
   private final String DEFAULT_SDK_VERSION;
```

Comment:
set the value here... 
```suggestion
  private final String DEFAULT_SDK_VERSION = "1.1.0";
```